powerpc: Export needed symbol for ZERO_PAGE usage in modules.

From: "Aneesh Kumar K.V" <aneesh.kumar@linux.vnet.ibm.com>

ext4 use ZERO_PAGE(0) to zero out blocks. We need to export
different symbols in different arch for the usage of ZERO_PAGE
in modules.

Signed-off-by: Aneesh Kumar K.V <aneesh.kumar@linux.vnet.ibm.com>
---
 arch/powerpc/kernel/ppc_ksyms.c |    2 ++
 1 file changed, 2 insertions(+)

Index: linux-2.6.25-rc6/arch/powerpc/kernel/ppc_ksyms.c
===================================================================
--- linux-2.6.25-rc6.orig/arch/powerpc/kernel/ppc_ksyms.c	2008-03-16 16:32:14.000000000 -0700
+++ linux-2.6.25-rc6/arch/powerpc/kernel/ppc_ksyms.c	2008-03-25 14:47:50.000000000 -0700
@@ -192,3 +192,5 @@ EXPORT_SYMBOL(intercept_table);
 EXPORT_SYMBOL(__mtdcr);
 EXPORT_SYMBOL(__mfdcr);
 #endif
+
+EXPORT_SYMBOL(empty_zero_page);
